packages/tools/devtools/devtools-view/src/components/ContainerSummaryView.tsx

```diff
@@ -320,7 +323,11 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 					<TableBody>
 						<DataRow
 							label="Status"
-							infoTooltipContent={containerStatusTooltipText}
+							infoTooltipContent={containerStatusTooltipText(
+								containerFeatureFlags.canModifyContainerState === false
+									? "Container Runtime"
+									: "Container",
```

Comment:
I think we should probably revert this change. I don't think this distinction matters in this context, and I think assuming "Container Runtime" because `canModifyContainerState` is false is very fragile.